### DISCUSSION TITLE
2.5.0怎么还不能更新

### DISCUSSION THREAD

**Discussion by RawDiamondMC** [Meloong-Git/PCL#1289] [提问] [answered]
github上release了一天了，还不能更新

**Comment by RawDiamondMC** [comment]
急急急急急，我是急急国王

**Comment by Pigeon0v0** [comment, marked as answer]
在 GitHub 上 Release 代表更新已经做完了，但可能还需要向杀毒软件提交样本请求信任啥的，别急

**Reply by RawDiamondMC** [reply]
> 在 GitHub 上 Release 代表更新已经做完了，但可能还需要向杀毒软件提交样本请求信任啥的，别急

快照版本也要提交样本请求？~毕竟我直接白名单的~

**Reply by LTCatt** [reply]
要提交的.jpg
